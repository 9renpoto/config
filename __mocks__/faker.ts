const faker = require.requireActual('faker')
const mockDate = require('mockdate')
faker.locale = 'ja'
mockDate.set('01/01/2020')
faker.seed(123)
module.exports = faker
