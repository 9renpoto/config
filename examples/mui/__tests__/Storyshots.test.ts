import initStoryshots from "@storybook/addon-storyshots";
import faker from "faker";

faker.seed(123);
initStoryshots({
  storyKindRegex: /^((?!.*?DontTest).)*$/
});
