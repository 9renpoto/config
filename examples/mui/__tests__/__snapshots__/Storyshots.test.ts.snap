// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots CardActionArea default 1`] = `
<button
  className="MuiButtonBase-root-4 MuiCardActionArea-root-1"
  onBlur={[Function]}
  onContextMenu={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex="0"
  type="button"
>
  <span
    className="MuiCardActionArea-focusHighlight-3"
  />
  <span
    className="MuiTouchRipple-root-7"
  />
</button>
`;
